config/routes.rb

```diff
@@ -42,7 +42,8 @@
       get "cancel_form"
       post "cancel"
       get "history"
-      get "revert_design" => "proposals#revert_design"
+      get "activate_design" => "proposals#activate_detail_design"
```

Comment:
We're going to remove these because we're not going to support that feature.

But also, using GETs to mutate data is a security flaw.